README.md

```diff
@@ -4,7 +4,10 @@ Kubernetes.
 [![Build Status](https://travis-ci.org/camsas/poseidon.svg)](https://travis-ci.org/camsas/poseidon)
 [![Coverage Status](https://coveralls.io/repos/github/camsas/poseidon/badge.svg?branch=master)](https://coveralls.io/github/camsas/poseidon?branch=master)
 
-***Note: this repo contains an initial prototype, it may break at any time! :)***
+***Note:
```

Comment:
@shivramsrivastava do you really want this PR? if not we can close this.  The repo should be buildable now.